### DISCUSSION TITLE
Why is this dependent on a custom fork of hydra?

### DISCUSSION THREAD

**Discussion by miversen33** [smoka7/multicursors.nvim#52] [General]
I am currently unable to use this plugin as I make extensive use of hydra in my configuration and this plugin cannot currently install due to a conflicting hydra version.

When I took at a look at the differences, I see a few small changes to the health check but nothing more. Is this something that can potentially be contributed back to the hydra project to avoid this sort of fragmentation?

I am not important but I feel like forcing people to use your own fork of a popular project will severely limit the reach your project has

**Comment by smoka7** [comment]
Thanks for sharing your concern, you can use the upstream for your dependency, It's not forced right now.
At first I had some issues with hydra and I had some commits to it to fix it, later I found a workaround so I didn't push those commits.
Since I wasn't sure that those kind of issues will not happen in the future I kept the fork.

>  Is this something that can potentially be contributed back to the hydra project to avoid this sort of fragmentation?

If i have something useful to hydra I definitely push upstream too.

**Comment by miversen33** [comment]
> Thanks for sharing your concern, you can use the upstream for your dependency, It's not forced right now.

This doesn't provide a sense of security that it wont be in the future. The project looks cool, best of luck with it :)

**Comment by MariaSolOs** [comment]
Pardon me for squeezing my opinion here, but I agree with @miversen33. I don't currently use `hydra` with any other plugin so I'm not experiencing any conflicts, but I did feel that it was a bit sus for this plugin to rely on a fork of a popular plugin. If the difference between the fork and the original repo increases, it might cause people to stop using this plugin.

@smoka7 Given that your fork fixes a harmless thing, I really don't see any gain from documenting your fork as the "official" dependency, and it is harming the power/quality of your plugin. Sure, there's a non-updated health check, but multicursor works just fine. If someone complains here about it you can ask them to reroute the issue to hydra instead, or provide your fork as a temporary solution.
